internal/broker/broker.go

```diff
@@ -310,6 +310,12 @@ func (b *Broker) authModeIsAvailable(session session, authMode string) bool {
 			return true
 		}
 
+		if session.isOffline {
+			// If the session is in offline mode, we can't register the device anyway,
+			// so we can allow the user to use local password authentication.
+			return true
```

Comment:
This does not imply that the user can access if the force-online option has been set, right?